GVRf/Framework/jni/objects/textures/compressed_texture_jni.cpp

```diff
@@ -30,7 +30,7 @@ Java_org_gearvrf_asynchronous_NativeCompressedTexture_normalConstructor(JNIEnv *
 
 JNIEXPORT jlong JNICALL
 Java_org_gearvrf_asynchronous_NativeCompressedTexture_mipmappedConstructor(JNIEnv * env,
-        jobject obj, jint target);
+        jobject obj, jint target, jintArray jtexture_parameters);
```

Comment:
I for one don't see anything wrong with passing the parameters to the ctor. The GLTexture ctor is applying default values for these parameters so it seems natural to me to be able to pass values of your choice also.